app/services/solr_mapper.rb

```diff
@@ -62,7 +62,8 @@ def call
       geo_place_ssim: geo_locations_field,
       provider_identifier_map_struct_ss: provider_identifiers_map.presence&.to_json,
       stanford_project_ssi: metadata['stanford_project'],
-      stanford_contributor_ssi: stanford_contributor?
+      stanford_contributor_ssi: stanford_contributor?,
+      stanford_dataset_ssi: metadata['stanford_project'] || stanford_contributor?
```

Comment:
I just realized this...shouldn't these be like `stanford_contributor_bsi` instead of `stanford_contributor_ssi`? We're storing booleans, not strings.

Reply:
Yeah, I thought it would be alright for now either way : ) I can change all three to be boolean in this PR